pex/http.py

```diff
@@ -134,6 +134,8 @@ def resolve(self, link):
 class UrllibContext(Context):
   """Default Python standard library Context."""
 
+  USER_AGENT = 'pex/%s' % PEX_VERSION
```

Comment:
This should be lifted up to the base class since RequestsContext already defines the same and we really want to always send a single UA no matter the Context impl.

Reply:
Hey, @jsirois , thank you. Moved to `Context`